addOns/pscanrulesAlpha/src/test/java/org/zaproxy/zap/extension/pscanrulesAlpha/FeaturePolicyScanRuleUnitTest.java

```diff
@@ -82,16 +83,28 @@ void shouldNotRaiseAlertOnMissingFeaturePolicyOthers() throws Exception {
     }
 
     @Test
-    void shouldNotRaiseAlertOnAvailableFeaturePolicy() throws Exception {
+    void shouldNotRaiseAlertOnAvailablePermissionsPolicy() throws Exception {
```

Comment:
All methods should be renamed to match the new name (or be generic, e.g. `shouldNotRaiseAlertOnAvailableHeader`)